app/agent/rest/host.js

```diff
@@ -0,0 +1,15 @@
+angular.module('portainer.agent').factory('Host', [
+  '$resource', 'API_ENDPOINT_ENDPOINTS', 'EndpointProvider',
+  function AgentFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider) {
+    'use strict';
+    return $resource(
+      API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/host/:action',
+      {
+        endpointId: EndpointProvider.endpointID
+      },
+      {
+        info: { method: 'GET', isArray: false, params: { action: 'info' } }
```

Comment:
No need to specify the `isArray` parameter to false here, it's the default value. You can remove it.